js/modules/loki_app_dot_net_api.js

```diff
@@ -494,12 +495,20 @@ class LokiAppDotNetServerAPI {
           fetchOptions
         ));
       } else {
+        // disable check for .loki
+        process.env.NODE_TLS_REJECT_UNAUTHORIZED = endpoint.match(/\.loki\//) ? 0 : 1;
```

Comment:
Doesn't using a custom agent work in this case?

Reply:
I had run tests where I thought it was working but when we merged it all yesterday, it stopped working. This at least prevents open groups from being invalid (when we don’t want them)

Reply:
And to be clear, this PR enforces TLS checks now disabled in the lokiRPC (from the fix clearnet PR).

And because I’m unsure if .loki will need this level or not. At least until I get clarity on what’s going on with nodes.

The custom agent only turns off some checks; it doesn’t affect all the checks the environmental variable version changes

Reply:
👍